spec/geostoreSpec.js

```diff
@@ -517,7 +517,7 @@ describe("geostore", function() {
         function (err, res) {
           expect(err).toEqual(null);
           expect(res.length).toEqual(1);
-          expect(res.id).toEqual(1);
+          expect(res[0].id).toEqual(1);
```

Comment:
Although this check has nothing to do with the new intersects logic, it was failing due to `res` being an Array vs checking the first item in `res`.